_overviews/scala3-scaladoc/blog.md

```diff
@@ -33,4 +33,29 @@ Scaladoc loads blog if the `_blog` directory exists.
 All the blogpost filenames should start with date in numeric format matching `YYYY-MM-DD`.
 Example name is `2015-10-23-dotty-compiler-bootstraps.md`.
 
+## Structure
 
+When creating a blog page, you will have a first part to fill in with different fields. See the following example:
+
+```
+---
+layout: <A reference to the layout page for the blog page>
+author: <Name of the author of the page>
+title: <Title of the page>
+subTitle: <Subtitle of the page>
+excerpt_separator: <!--more-->
```

Comment:
This doesn't really explain what this is used for.

Reply:
I just saw that it is no longer used in the latest version. Maybe an issue or deliberately done.